README.md

```diff
@@ -22,6 +18,34 @@ This guide will show you how to:
 
 [Try out the example.](https://alphagov.github.io/govuk-country-and-territory-autocomplete/examples/)
 
+---
+
+## Support
+
+The GOV.UK Design System team maintains the country and territory autocomplete. However, we’re only able to put in minimal work to support it.
+
+[Read about our plans to maintain this component.](LINK-TO-PINNED-RESEARCH-ISSUE)
+
+We provide support for:
```

Comment:
Might not want to use 'provide' here, if it implies we can only help users to fix these things, rather than fixing them ourselves. [See comment on other autocomplete PR for more context](https://github.com/alphagov/accessible-autocomplete/pull/525/files#r785790989).